pupa/settings.py

```diff
@@ -9,6 +9,10 @@
 
 # scrape settings
 
+ENABLE_KAFKA = os.environ.get('ENABLE_KAFKA', "False").lower() == "true"
+KAFKA_REPORT_TOPIC = 'post-scrape-reports'
```

Comment:
In order to forward the scraper, are we going to listen to all of those streams? Don't you have to manually create streams?

I say we punt until after this becomes an issue. Also custom pupa stuff can change the name via `settings.`. I think this is good for a proof of concept. Nothing has to be final.